mjolnir_snap/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: mjolnir
+summary: Mjolnir snap
+description: A moderation tool for Matrix
+version: "1.6.4"
+base: core22
+license: Apache-2.0
+confinement: strict
+apps:
+  mjolnir:
+    command: 'bin/node $SNAP/mjolnir/index.js'
```

Comment:
Nit - on line 7 you have double quotes and here it is single quotes, could it be consistent?